app/base-env/Dockerfile

```diff
@@ -64,7 +64,9 @@ RUN curl -L https://github.com/github/codeql-action/releases/download/codeql-bun
 ## LLVM/GCC SETUP ##
 ####################
 ADD ./app/base-env/llvm.sh ./app/base-env/gcc.sh /
-RUN /llvm.sh ${CLANG_VERSION} && /gcc.sh ${GCC_VERSION}
+ARG FORCE_LLVM_INSTALL="OFF"
+RUN if [ "$CC" == clang* ] || [ ${FORCE_LLVM_INSTALL} == "ON" ]; then /llvm.sh ${CLANG_VERSION}; fi
```

Comment:
I would rather have an opt-out rather than an opt-in here:
* it's nice to have clang-format/clang-tidy in all the images
* build:coverage depends on clang-format